crates/hud/src/draft.rs

```diff
@@ -0,0 +1,19 @@
+use bevy::prelude::{Commands, info, Transform, Vec3};
+
+use de_core::objects::BuildingType;
+use de_spawner::DraftBundle;
+
+pub fn place_draft(
```

Comment:
Pretty much the same thing is already implemented inside controller (when P or B is pressed). I suggest to create an event for this inside controller and send that event instead (from both the keyboard handling system in controller and the button handling system in HUD).

This deserves a bit more thought to avoid a circular dependency (between hud and controller). Maybe the two should be merged together? What do you think?

Reply:
I do agree, I have created new [PR](https://github.com/DigitalExtinction/Game/pull/334) for that